pattern-library/js/edx-icons.js

```diff
@@ -1,9 +1,13 @@
-define(['afontgarde'],
-    function() {
-        'use strict';
+define(['afontgarde'], function() {
+    'use strict';
 
-        window.AFontGarde('edx-icons', {
-            // Check a few random icons to see if our font loads
-            glyphs: '\uE621\uE622\uE623'
-        });
+    // Old icon fonts, currently used in the platform.
+    window.AFontGarde('FontAwesome', {
```

Comment:
So yes, I do think you should have an edx-platform specific version that lives in that repo. It could import this one and then override it, or it could just redefine it from scratch.